gui-tests/src/test/kotlin/tanvd/grazi/ide/ui/EditorGuiTest.kt

```diff
@@ -0,0 +1,117 @@
+package tanvd.grazi.ide.ui
+
+import com.intellij.lang.annotation.HighlightSeverity
+import com.intellij.testGuiFramework.fixtures.JDialogFixture
+import com.intellij.testGuiFramework.impl.*
+import com.intellij.testGuiFramework.util.step
+import org.junit.Test
+import tanvd.grazi.GraziGuiTestBase
+
+class EditorGuiTest : GraziGuiTestBase() {
+    private fun JDialogFixture.toggleTestRules() {
+        fun toggle(vararg path: String) {
+            with(checkboxTree(*path)) {
+                robot().moveMouse(0, 0)
+                clickCheckbox()
+            }
+        }
+
+        toggle("Russian", "Заглавные буквы", "Все буквы в слове ЗАГЛАВНЫЕ")
```

Comment:
Let's divide it in `step`-s as other tests

Reply:
Added some steps